projects/1.18/assets/andycats-flowers/andycats_flowers/lang/zh_cn.json

```
@@ -0,0 +1,141 @@
+{
+  "block.andycats_flowers.cyan_cosmos": "青色大波斯菊",
+  "block.andycats_flowers.purple_gladiolus": "紫色剑兰",
```

```suggestion
  "block.andycats_flowers.purple_gladiolus": "紫色唐菖蒲",
```
仅学名建议，剑兰作为俗名应该也OK吧
http://www.iplant.cn/info/Gladiolus%20gandavensis

---

面向普通玩家还是建议使用俗名，学名未必大家都知道